nautilus_trader/adapters/bybit/endpoints/trade/batch_place_order.py

```diff
@@ -29,16 +29,21 @@
 from nautilus_trader.core.nautilus_pyo3 import HttpMethod
 
 
-class BybitBatchPlaceOrder(msgspec.Struct, omit_defaults=True, frozen=True):
+class BybitBatchPlaceOrder(
```

Comment:
Do we intend on using all of these fields? or was the goal here to exactly match the API spec?

Reply:
I compared the fields one by one with those in the documentation.

https://bybit-exchange.github.io/docs/v5/order/create-order